### PR TITLE
BE: Events: Parse event label

### PR DESCRIPTION
**Pull request by Mohammad-Junid**
## Description
Currently we are not ignoring event label and not parsing from event conf. we should parse event label, persist it and propagate it to Upstream in BFF and to UI

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2450

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Comment by Mohammad-Junid** [comment]
image

**Review by aurang-zeb313 — APPROVED** [review]
LGTM!

File: shared-lib/event-conf/src/main/java/org/opennms/horizon/events/xml/Event.java [hunk 1]
```diff
@@ -268,6 +268,12 @@ public class Event implements Serializable {
     @Valid
     private AlertData _alertData;
 
+    @XmlElement(name = "severity", required = true)
+    private String _severity;
```
[the remark this answers is not in the document]
Comment (Mohammad-Junid) [reply shared-lib/event-conf/src/main/java/org/opennms/horizon/events/xml/Event.java]: ( _  )  are removed from "_severity" as well as from  "_evetLabel" fileds

File: shared-lib/events/src/main/proto/events.proto [hunk 2]
```diff
@@ -100,3 +102,35 @@ service EventService {
   rpc getEventsByNodeId(google.protobuf.UInt64Value) returns (EventLog) {};
   rpc searchEvents(EventsSearchBy) returns (EventLogListResponse) {};
 }
+
+enum Severity {
+  // No or missing value
+  SEVERITY_UNDEFINED = 0;
```
[the remark this answers is not in the document]
Comment (Mohammad-Junid) [reply shared-lib/events/src/main/proto/events.proto]: Changes are done

**Review by smunir-onms — COMMENTED** [review]
LGTM, discussed changes verified.

**Review by fooker — APPROVED** [review]

**Pull request merged**